spec/embulk/input/jira_api/client_spec.rb

```diff
@@ -67,15 +69,23 @@
       ]
     end
 
-    subject { Embulk::Input::JiraApi::Client.new.search_issues(jql) }
+    subject { jira_api.search_issues(jql) }
 
-    it do
+    it "Search issues successfully" do
       allow(Jiralicious).to receive_message_chain(:search, :issues_raw).and_return(results)
       allow(Jiralicious::Issue).to receive(:find).and_return(results.first)
 
       expect(subject).to be_kind_of Array
       expect(subject.map(&:class)).to match_array [Embulk::Input::JiraApi::Issue, Embulk::Input::JiraApi::Issue]
     end
+
+    it "Search issues got 401 due to high concurrent load issues" do
```

Comment:
Could you add another test to reflect new logic (dynamic concurrent threads), ie. partial error?

Reply:
Done